spec/rspec/matchers/built_in/be_a_subclass_of_spec.rb

```diff
@@ -0,0 +1,45 @@
+SuperKlass = Class.new
+SubKlass = Class.new(SuperKlass)
+SubSubKlass = Class.new(SubKlass)
+OtherKlass = Class.new
+
+module RSpec
+  module Matcher
+    [:be_a_subclass_of, :be_subclass_of].each do |method|
+      RSpec.describe "expect(actual).to #{method}(expected)" do
+
+        it_behaves_like "an RSpec matcher", :valid_value => SubKlass, :invalid_value => OtherKlass do
+          let(:matcher) { send(method, SuperKlass) }
+        end
+
+        it "passes if actual is subclass of expected class" do
+          expect(SubKlass).to send(method, SuperKlass)
+        end
+
+        it "passes if actual is subklass of subclass of expected class" do
+          expect(SubSubKlass).to send(method, SuperKlass)
+        end
+
+        it "fails with failure message for should unless actual is subclass of expected class" do
```

Comment:
The "for should" part feels out of place since that's not the main syntax anymore.  In fact, I wouldn't even mention the failure message in the doc string here (but keep it in the expectation below).